src/test/java/com/codurance/retropolis/utils/MockMvcWrapper.java

```diff
@@ -30,9 +30,10 @@ public static HttpHeaders getAuthHeader(String token) {
     return headers;
   }
 
-  public String postRequest(String url, String content, ResultMatcher response) throws Exception {
+  public String postRequest(String url, String content, ResultMatcher response, HttpHeaders headers) throws Exception {
     return mockMvc.perform(MockMvcRequestBuilders.post(url)
         .content(content)
+        .headers(headers)
```

Comment:
This allows us to use the headers that we have set in: https://github.com/codurance/Retropolis-BE/pull/41/files#diff-885168c7919a863756e128fe6d443e2d